### PR TITLE
Capstone project: Fix path and syntax issues

### PR DESCRIPTION
**Pull request by ashutoshpatel2210**
Added paths and Fixed the issues with code

### CODE REVIEW

File: Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py [hunk 1]
```diff
@@ -100,35 +101,39 @@ def get_classification(self, image):
...
         # Filter boxes with a confidence score less than `confidence_cutoff`
-        final_boxes, final_scores, final_classes = filter_boxes(confidence_cutoff, boxes, scores, classes)
-        
+        final_boxes, final_scores, final_classes = self.filter_boxes(confidence_cutoff, boxes, scores, classes)
         # Enable for Visualization
-        #width, height = image.size
-        #box_coords = to_image_coords(final_boxes, height, width)
-        #draw_boxes(image, box_coords, classes)
-        
+        if (self.enable_visualization):
```
Comment (zackZhu1) [inline Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py]: No need to have left and right parenthesis in if statement

File: Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py [hunk 2]
```diff
@@ -100,35 +101,39 @@ def get_classification(self, image):
...
+            cd
         
         if final_scores[0] is not None and final_scores[0] > confidence_cutoff:
             class_name = self.category_index[final_classes[0]]['name']
             #print((class_name, final_scores[0]))
+            rospy.logwarn("final_class_prediction={}".format(final_classes[0]))
             if (final_classes[0] == 1):
                 return TrafficLight.GREEN
-            elif (final_classes[0] == 2:
+            elif (final_classes[0] == 2):
```
Comment (zackZhu1) [inline Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py]: same here

File: Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py [hunk 3]
```diff
@@ -100,35 +101,39 @@ def get_classification(self, image):
...
                 return TrafficLight.YELLOW
-            elif (final_classes[0] == 3:
+            elif (final_classes[0] == 3):
```
Comment (zackZhu1) [inline Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py]: same here

File: Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py (hunk 1 above)
Comment (ashutoshpatel2210) [reply Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py]: fixed

File: Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py (hunk 2 above)
Comment (ashutoshpatel2210) [reply Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py]: fixed

File: Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py (hunk 3 above)
Comment (ashutoshpatel2210) [reply Self_Driving_Car_Engineer/Capstone_Project/CarND-Capstone/ros/src/tl_detector/light_classification/tl_classifier.py]: fixed

**Review by arunsuresh16 — APPROVED** [review]

**Pull request merged**